src/customize-pop/application-notifications.md

```diff
@@ -0,0 +1,31 @@
+# Application Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notification pop-ups globally. Notifications will still queue up in the notiications drop-down while this setting is enabled. |
+| Lock Screen Notifications | Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options. |
+
+## Viewing Notifications
+
+Application notifications appear in the upper center of the screen. Click the date and time to view recent notifications. You can also globally disable notifications by toggling `Do Not Disturb`.
```

Comment:
> Application notifications appear in the upper center of the screen.

Maybe add "by default" to the end of that sentence, since this can be configured and that was covered earlier.

Reply:
Added